static/app/components/group/suggestedOwners/suggestedOwners.tsx

```diff
@@ -186,16 +182,15 @@ class SuggestedOwners extends AsyncComponent<Props, State> {
 
 function SuggestedOwnersWrapper(props: Omit<Props, 'committers' | 'organization'>) {
   const organization = useOrganization();
-  const {committers, releaseCommitters} = useCommitters({
+  const {data} = useCommitters({
```

Comment:
not a real issue in this case, but just FYI if you don't care about the loading state you can pass `notifyOnChangeProps: ['data']` to prevent this hook from rerendering when the loading state changes. I like setting up my hooks so you can pass the options object as a second argument.